### PR TITLE
Make FS error more clear.

### PR DESCRIPTION
**Pull request by anthonyshew**

### CODE REVIEW

**Review by chris-olszewski — APPROVED** [review]

File: crates/turborepo-lib/src/commands/prune.rs [hunk 1]
```diff
@@ -24,7 +24,7 @@ pub const DEFAULT_OUTPUT_DIR: &str = "out";
 pub enum Error {
     #[error("io error while pruning: {0}")]
     Io(#[from] std::io::Error),
-    #[error("fs error while pruning: {0}")]
+    #[error("File system error while pruning. The error from the operating system was: {0}")]
```
[the remark this answers is not in the document]
Comment (anthonyshew) [reply crates/turborepo-lib/src/commands/prune.rs:27]: Present tense for sure.

Comment (anthonyshew) [inline crates/turborepo-lib/src/commands/prune.rs:27]:
```suggestion
    #[error("File system error while pruning. The error from the operating system is: {0}")]
```

File: examples/kitchen-sink/packages/ui/src/link/index.test.tsx [hunk 2]
```diff
@@ -1,11 +0,0 @@
-import { createRoot } from "react-dom/client";
```
Comment (tknickman) [inline examples/kitchen-sink/packages/ui/src/link/index.test.tsx:1]: Are these tests intentional removals?

File: crates/turborepo-lib/src/commands/prune.rs (hunk 1 above)
Comment (anthonyshew) [inline crates/turborepo-lib/src/commands/prune.rs:27]:
```suggestion
    #[error("File system error while pruning. The error from the operating system is: {0}")]
```

**Pull request merged**